ftw/linkchecker/linkchecker.py

```diff
@@ -0,0 +1,100 @@
+from multiprocessing import Pool
+import time
+import requests
+import urls_testing
+import os
+import sys
+
+urls = urls_testing.urls
+number_of_processes = len(urls)
+timeout = 1
+
+
+def millis():
+    return int(round(time.time() * 1000))
+
+
+def get_uri_responses(urls):
```

Comment:
In this file, the name `urls`  is used for different purposes:
- the list of url-items
- one single url-item containing source and target, like on this line
I'd probably name one item "link" and let it contain source and target urls. This could improve the readibility of the code.

Reply:
👍